zulipterminal/ui_tools/boxes.py

```diff
@@ -1129,9 +1129,8 @@ def soup2markup(cls, soup: Any, metadata: Dict[str, Any],
                 elif tag == 'div' and any(
                         _ in css_classes for _ in unrendered_div_classes):
                     # UNRENDERED DIV CLASSES
-                    matching_class = (set(unrendered_div_classes)
-                                      & set(css_classes))
-                    text = unrendered_div_classes[matching_class.pop()]
+                    match = set(unrendered_div_classes) & set(css_classes)
+                    text = unrendered_div_classes[match.pop()]
```

Comment:
This seems like something we could include in that new simplifying commit, though I'd suggest `matches` since that's more general. We should likely have a note here about the (unlikely) potential for multiple matches.

Reply:
:+1: